main.go

```diff
@@ -345,18 +344,18 @@ func main() {
 		}()
 	}
 
-	var registerClient = k8s.AdmissionWebhookRegisterClient{
-		Logger: logger.Named("admissionWebhookRegisterClient"),
-	}
+	// var registerClient = k8s.AdmissionWebhookRegisterClient{
+	// 	Logger: logger.Named("admissionWebhookRegisterClient"),
+	// }
 
-	err = registerClient.Register(ctx, conf)
-	if err != nil {
-		prod.Fatal(err.Error())
-	}
+	// err = registerClient.Register(ctx, conf)
```

Comment:
Does this cause any problem if we dont comment it out? Since this is opensource code, we would want to upstream all our changes to the nsm github project. So we should not be making changes that are not compatible with the upstream.

Reply:
So if we don't avoid calling `registerClient.Register` function it will create the webhook config and now there will be two webhook configs one created by this function and another created by helm. And other issue I am facing is that the gw pod is coming up also.

So can we set an ENV in the helm chart in webhook deployment and if that is present then we can avoid calling this function.